src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -215,6 +215,34 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(boolean sessionI
             tableBodyHtml.append(tableRow + Const.EOL);
         }
         
+        StringBuilder mobileHtml = new StringBuilder();
```

Comment:
The function getQuestionWithExistingResponseSubmissionFormHtml looks very long now. Can refactor it into smaller functions to make it neater.